app/views/pages/api_release_notes.html.haml

```diff
@@ -1347,131 +1188,95 @@
-          = govuk_table_td('data-label': "Action") do
-            = 'Create advocate claim'
-          = govuk_table_td('data-label': "Endpoint URL") do
-            = 'POST /api/external_users/claims'
-        = govuk_table_row do
-          = govuk_table_td('data-label': "Action") do
-            = 'Validate litigator final claim'
-          = govuk_table_td('data-label': "Endpoint URL") do
-            = 'POST /api/external_users/claims/final/validate'
-        = govuk_table_row do
-          = govuk_table_td('data-label': "Action") do
-            = 'Create litigator interim claim'
-          = govuk_table_td('data-label': "Endpoint URL") do
-            = 'POST /api/external_users/claims/final'
-        = govuk_table_row do
-          = govuk_table_td('data-label': "Action") do
-            = 'Validate litigator interim claim'
-          = govuk_table_td('data-label': "Endpoint URL") do
-            = 'POST /api/external_users/claims/interim/validate'
-        = govuk_table_row do
-          = govuk_table_td('data-label': "Action") do
-            = 'Create litigator interim claim'
-          = govuk_table_td('data-label': "Endpoint URL") do
-            = 'POST /api/external_users/claims/interim'
-        = govuk_table_row do
-          = govuk_table_td('data-label': "Action") do
-            = 'Validate litigator transfer claim'
-          = govuk_table_td('data-label': "Endpoint URL") do
-            = 'POST /api/external_users/claims/transfer/validate'
-        = govuk_table_row do
-          = govuk_table_td('data-label': "Action") do
-            = 'Create litigator transfer claim'
-          = govuk_table_td('data-label': "Endpoint URL") do
-            = 'POST /api/external_users/claims/transfer'
+      = govuk_table(classes: 'app-table--responsive') do |table|
+        = table.with_head do |head|
+          = head.with_row do |row|
+            = row.with_cell(text: 'Action')
+            = row.with_cell(text: 'Endpoint URL')
+        = table.with_body do |body|
+          = body.with_row do |row|
+            = row.with_cell(html_attributes: { 'data-label': "Action" }, text: 'Validate advocate claim')
+            = row.with_cell(html_attributes: { 'data-label': "Endpoint URL" }, text: 'POST /api/external_users/claims/validate')
+          = body.with_row do |row|
+            = row.with_cell(html_attributes: { 'data-label': "Action" }, text: 'Create advocate claim')
+            = row.with_cell(html_attributes: { 'data-label': "Endpoint URL" }, text: 'POST /api/external_users/claims')
+          = body.with_row do |row|
+            = row.with_cell(html_attributes: { 'data-label': "Action" }, text: 'Validate litigator final claim')
+            = row.with_cell(html_attributes: { 'data-label': "Endpoint URL" }, text: 'POST /api/external_users/claims/final/validate')
+          = body.with_row do |row|
+            = row.with_cell(html_attributes: { 'data-label': "Action" }, text: 'Create litigator final claim')
```

Comment:
Hi @mpw5 I am not sure if this is worth pointing out but I can't find `Create litigator final claim' in the old code? Just wanted to check it was added intentionally. :)

Reply:
Thanks Mark - that is a good spot! :)